module-utils/utility/Anonymize.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <string>
+#include <algorithm>
+#include <functional>
+#include <map>
+
+namespace utils::anonymize
+{
+    // Anonymized text will replace by this character
+    constexpr auto anonymizationCharacter = '*';
```

Comment:
Move all constexpr variables which are used **only** in Anonymize to the anonymous namespace in .cpp
They don't need to be available everywhere in the code if they are only required in those functions